packages/core/src/render3/instructions/instructions.ts

```diff
@@ -1141,7 +1168,42 @@ export function elementAttribute(
   }
 }
 
+// TODO: Remove this when the issue is resolved.
+/**
+ * Tsickle freaked out over a function returning itself
```

Comment:
Can we be a bit more precise than "freaked out"? 

```suggestion
 * Tsickle has a bug where it creates an infinite loop for a function returning itself.
 * This is a temporary type that will be removed when the issue is resolved.
```